docs/api/session.md

```diff
@@ -547,3 +547,24 @@ The `listener` will be called with `listener(details)` when an error occurs.
   * `timestamp` Double
   * `fromCache` Boolean
   * `error` String - The error description.
+
+#### `ses.protocol`
+
+Returns an instance of [protocol](protocol.md) module for this session.
+
+```javascript
+const electron = require('electron');
+const app = electron.app;
```

Comment:
We can use destructuring assignment here:

```js
const {app, session} = require('electron')
```

And for new code samples in docs we should use new coding style, e.g. without `;`.